src/include/storage/buffer_manager/memory_manager.h

```diff
@@ -69,12 +84,18 @@ class KUZU_API MemoryManager {
 
     BufferManager* getBufferManager() const { return bm; }
 
+    void addUnusedChunk(ChunkedNodeGroup* nodeGroup);
+    void clearUnusedChunk(ChunkedNodeGroup* nodeGroup);
+    void loadFromDisk(ChunkedNodeGroup* nodeGroup);
```

Comment:
Are these functions defined and used anywhere?

Reply:
One of them was defined, but no, those should have been removed. I'd initially implemented this stuff in the memory manager before realising that it needed to be better integrated into the buffer manager.